tests/plugins/uploadwidget/manual/imagecustomtypes.html

```diff
@@ -0,0 +1,15 @@
+<div style="display: none; text-align: center; color: red; font-weight: bold; padding: 20px;" id="ignore">
```

Comment:
It's more test for uploadimage, not uploadwidget, therefore it should be moved to `tests/plugins/uploadimage/manual`.